newrelic.js

```diff
@@ -44,6 +44,38 @@ exports.config = {
    * attributes include/exclude lists.
    */
   allow_all_headers: true,
+  application_logging: {
+    /**
+     * If this is disabled, all sub-features are disabled.
+     * If it is enabled, the individual sub-feature configurations take effect.
+     */
+    enabled: true,
+    forwarding: {
+      /**
+       * Toggles whether the agent gathers log records for sending to New Relic.
+       */
+      enabled: false,
+      /**
+       * Number of log records to send per minute to New Relic.
+       * Controls the overall memory consumption when using log forwarding.
+       */
+      max_samples_stored: 10000
```

Comment:
we don't intend for users to change this and if we do they can set it themselves.  I recommend re-reading the ticket